pkg/config/rdbms.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+// RDBMSConfig specifies the properties required to use an RDBMS as the storage backend
+type RDBMSConfig struct {
+	Name    string `validate:"required" envconfig:"ATHENS_RDBMS_STORAGE_NAME"`
+	Timeout int
```

Comment:
could rename it to `TimeoutInMs` with the duration unit so we know the usage.

Reply:
It's in seconds and derived from the global timeout currently.

Reply:
so renaming to`TimeoutSec` would help.

Reply:
That would be another PR, we do intend to have more than the one timeout. If there is not an issue open yet, can you please open one for that? This PR however should just move us from where we are to using a config file.

Reply:
relevant to this https://github.com/gomods/athens/issues/261